konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/format/JavaNDArrays.java

```diff
@@ -88,5 +88,591 @@ public Float5Array(float[][][][][] array) {
         }
     }
 
+    private static abstract class BaseDoubleArray<T> extends BaseNDArray<T>{
+        protected final long[] shape;
+        public BaseDoubleArray(T array, long[] shape) {
+            super(array);
+            this.shape = shape;
+        }
+
+        @Override
+        public NDArrayType type() {
+            return NDArrayType.DOUBLE;
+        }
+
+        @Override
+        public long[] shape() {
+            return shape;
+        }
+    }
+
+    public static class Double1Array extends BaseDoubleArray<double[]>{
+        public Double1Array(double[] array) {
+            super(array, new long[]{array.length});
+        }
+    }
+
+    public static class Double2Array extends BaseDoubleArray<double[][]>{
+        public Double2Array(double[][] array) {
+            super(array, new long[]{array.length, array[0].length});
+        }
+    }
+
+    public static class Double3Array extends BaseDoubleArray<double[][][]>{
+        public Double3Array(double[][][] array) {
+            super(array, new long[]{array.length, array[0].length, array[0][0].length});
+        }
+    }
+
+    public static class Double4Array extends BaseDoubleArray<double[][][][]>{
+        public Double4Array(double[][][][] array) {
+            super(array, new long[]{array.length, array[0].length, array[0][0].length, array[0][0][0].length});
+        }
+    }
+
+    public static class Double5Array extends BaseDoubleArray<double[][][][][]>{
+        public Double5Array(double[][][][][] array) {
+            super(array, new long[]{array.length, array[0].length, array[0][0].length, array[0][0][0].length, array[0][0][0][0].length});
+
+        }
+    }
+
+    private static abstract class BaseBfloat16Array<T> extends BaseNDArray<T>{
```

Comment:
BFloat16 and Float16 are not java types, so we shouldn't have Java NDArray for them.
So let's remove all of the BFloat16 and Float16 classes here.